sample-gominer.conf

```diff
@@ -38,4 +68,23 @@
 ; Mining settings
 ; ------------------------------------------------------------------------------
 
+; Location of kernel to use for mining.
+; kernel=./blake256.cl
+
+; Intensity (the work size is 2^intensity) per device
 ; intensity=26
+
+; Worksize sizes of the work to do per device (overrides intensity).
+; worksize=33554176
```

Comment:
Might want to double these up to make it a bit more clear that it can be specified multiple times.